lms/views/admin/email.py

```diff
@@ -0,0 +1,45 @@
+from datetime import datetime
+
+from pyramid.httpexceptions import HTTPFound
+from pyramid.view import view_config, view_defaults
+
+from lms.security import Permissions
+from lms.services import DigestService, MailchimpService
+
+
+@view_defaults(route_name="admin.email", permission=Permissions.ADMIN)
+class AdminEmailViews:
+    def __init__(self, request):
+        self.request = request
+        self.digest_service = request.find_service(DigestService)
+        self.mailchimp_service = request.find_service(MailchimpService)
+
+    @view_config(
+        route_name="admin.email",
+        request_method="GET",
+        renderer="lms:templates/admin/email.html.jinja2",
+    )
+    def get(self):
+        return {}
+
+    @view_config(route_name="admin.email", request_method="POST")
+    def post(self):
+        to_email = self.request.POST["to_email"]
+        h_userids = self.request.POST["h_userids"].split()
+
+        # Caution: datetime.fromisoformat() doesn't support all ISO 8601
+        # strings! This only works for the subset of ISO 8601 produced by
+        # datetime.isoformat().
+        since = datetime.fromisoformat(self.request.POST["since"])
+        until = datetime.fromisoformat(self.request.POST["until"])
```

Comment:
Not bothering with proper validation as of yet. I'd say it's not really necessary for an admin page like this one anyway, and haste is the priority for now.